kolibri/core/assets/src/views/userAccounts/__tests__/GenderSelect.spec.js

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/vue';
+import VueRouter from 'vue-router';
+import GenderSelect from '../GenderSelect.vue';
+import '@testing-library/jest-dom';
+
+const renderComponent = () => {
+  return render(GenderSelect, {
+    routes: new VueRouter(),
+  });
+};
+
+describe('GenderSelect', () => {
+  test('renders correctly with label placeholder and options', async () => {
+    renderComponent();
+
+    expect(screen.getByText('Gender')).toBeInTheDocument();
```

Comment:
I think we could use getByLabel, to diferentiate that this is the label of the input. And we can add an array of options that it should contain.

Also, the component is responsible for emitting events when a selected value changes, which could be another test case.

Reply:
Using `getByLabelText` does not work for this particular component. I suspect it's due to the markup that's surrounding it.
```html
<div class="ui-select-label-text is-inline">
   Gender
</div>
```